core/trino-main/src/test/java/io/trino/operator/scalar/TestVersionFunction.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.operator.scalar;
+
+import org.testng.annotations.Test;
+
+import static io.trino.spi.type.VarcharType.VARCHAR;
+
+public class TestVersionFunction
+        extends AbstractTestFunctions
+{
+    @Test
+    public void testVersion()
+    {
+        assertFunction("version()", VARCHAR, "<unknown>");
```

Comment:
This seems to be the only test and so we do not really know whether the function works or always returns "<unknown>".
Do we need a product test for this function?

Reply:
@findepi Added a product test as https://github.com/trinodb/trino/pull/4629/files#diff-04e9af1f77428c07efae26b7d40ed64059d1abcad3115dc97d25c23256e55773. Could you review it again?